Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/VanillaMemeEdit.java

```diff
@@ -22,6 +38,40 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    public void onShareClick(View v){
```

Comment:
^^^Disregard my comment i see its beneath it lol sorry its hard to read code on mobile